### PR TITLE
Fix `pkg/interop` commits in `go.mod`

### LINKED ISSUE #2392: `go get` error when updating neo-go
I tried to update `neo-go` for nspcc-dev/neofs-contract repo and caught this error:
```
> go get github.com/nspcc-dev/neo-go@master
go: downloading github.com/nspcc-dev/neo-go v0.98.2-pre.0.20220309155724-623fdee97c2d
go: downloading github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20220301152507-4c415683e74e
go: upgraded github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20220120102126-25583f9aeb13 => v0.0.0-20220301152507-4c415683e74e: error finding sum for github.com/nspcc-dev/neo-go/pkg/interop@v0.0.0-20220301152507-4c415683e74e: github.com/nspcc-dev/neo-go/pkg/interop@v0.0.0-20220301152507-4c415683e74e: invalid version: unknown revision 4c415683e74e
```

https://github.com/nspcc-dev/neo-go/search?q=4c415683e74e

I think we can extend check from 6b152daf7e428e4a to check something like `git merge-base HEAD <modules-commit> == <modules-commit>`, which will help for rebases. Or we can think again about how to organize the code to minimize probability of an error.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #2392 .

### CODE REVIEW

File: scripts/check_deps.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```
Comment (roman-khimov) [inline scripts/check_deps.sh:1]: sh?

File: scripts/update_deps.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (roman-khimov) [inline scripts/update_deps.sh:1]: sh?

File: scripts/check_deps.sh (hunk 1 above)
Comment (AnnaShaleva) [reply scripts/check_deps.sh:1]: Do we need it to work on Windows?

Comment (roman-khimov) [reply scripts/check_deps.sh:1]: I don't think it's a requirement. There are just helpers anyway.

File: scripts/update_deps.sh (hunk 2 above)
Comment (fyrchik) [reply scripts/update_deps.sh]: Fixed.

**Review by AnnaShaleva — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**